onnxruntime/contrib_ops/cuda/layer_norm_impl.cu

```diff
@@ -355,19 +356,66 @@ void HostApplyLayerNorm(
   const uint64_t maxGridY = prop.maxGridSize[1];
   const int warp_size = prop.warpSize;
   ORT_ENFORCE(warp_size == GPU_WARP_SIZE);
+  
+  bool custom_supported_dim = std::find(std::begin(SUPPORTED_REDUCTION_DIM), std::end(SUPPORTED_REDUCTION_DIM), n2) != std::end(SUPPORTED_REDUCTION_DIM);
+  bool supported_half2 = (prop.major >= 7) || !(std::is_same<T, half>::value);
+  if(custom_supported_dim && supported_half2){
+    launch_custom_layer_norm<T,U>(
+      output,
+      input,
+      gamma,
+      beta,
+      U(epsilon),
+      n1,
+      n2,
+      invvar,
+      mean);
+  }
+  else{
```

Comment:
please run formatter on updated files

Reply:
looks like this file was missed. also onnxruntime/test/contrib_ops/layer_norm_test.cc

Reply:
Ran on all updated files now, thanks!